view-user/src/components/common/Header/Header.tsx

```diff
@@ -1,24 +1,30 @@
 import React, { ReactNode } from "react";
 import styles from "./Header.module.css";
 import { useRouter } from "next/router";
+import { IoHelpCircleOutline } from "react-icons/io5";
 
-interface HeaderProps {
-  children: ReactNode;
-  user: string;
-}
-
-const Header = ({ children, user }: HeaderProps) => {
+const Header = () => {
   const router = useRouter();
   return (
     <div className={styles.container}>
       <div className={styles.main}>
-        <div className={styles.title} onClick={() => router.push("/")}>
-          <p>NUTFES BINGO {user}</p>
-        </div>
-        <div>{children}</div>
+        <img
+          className={styles.logo}
+          src="./Bingo_logo.png"
+          alt="sample"
+          onClick={func}
+        />
+        <button className={styles.icon}>
+          <IoHelpCircleOutline />
+        </button>
       </div>
     </div>
   );
 };
 
+// オンクリックの動作確認用の関数（後でヘルプに飛ぶようにする）
+var func = function () {
```

Comment:
```suggestion
const handleClick = () => {
```
極力varを使わないでconstで定義するようにしてください。
また関数の名称は分かりやすいものにしてください。
funcってなんですか？

Reply:
関数置いている位置がちょっとちがう。
useRouterの下に置いてほしいな
returnが一番最後にしてください